src/main/java/org/apache/maven/shared/dependency/analyzer/asm/ConstantPoolParser.java

```diff
@@ -168,20 +174,46 @@ static Set<String> parseConstantPoolClassReferences( ByteBuffer buf )
                     break;
             }
         }
+        
         Set<String> result = new HashSet<>();
+        
         for ( Integer aClass : classes )
         {
-            String className = stringConstants.get( aClass );
-
-            // filter out things from unnamed package, probably a false-positive
-            if ( isImportableClass( className ) )
+            addClassToResult( result, stringConstants.get( aClass ) );
+        }
+        
+        for ( Integer aType : types )
```

Comment:
nit: "type" would typically be used in java

Reply:
I assume you mean the loop variable name. I used `aType` because the other loop over the classes uses `aClass`. Wouldn't have been my choice either.

Reply:
There's a lot of idiosyncracy scattered through our codebase I'm trying to stamp out. aClass is only because class is a reserved word in Java, though in practice clazz is more commonly used. In either case, though, this is not necessary for type.

Reply:
I pushed a change renaming `classes`to `classReferences` and `aClass` to `classRef` and the same for `types`/`aType`.

Reply:
Mainly because I use `type` for another variable already ;)